scripts/release

```diff
@@ -2,7 +2,8 @@
 # This script updates lepidopter version and ChangeLog files
 set -e
 
-VERSION_FILE="lepidopter-fh/etc/lepidopter_version"
+source conf/lepidopter-image.conf
+UPDATE_INCREMENT=$(awk '{print $1+1}' ${UPDATE_VERSION_FILE})
```

Comment:
This [diff](https://github.com/TheTorProject/lepidopter/pull/97/files#diff-1d889b31bb06cb578abde8e2253d043aR89) (hopefully) ensures that the latest version of the updater is in the lepidopter image.
@hellais Do you still think that it's not robust enough?